### PR TITLE
Update Redux commit hash

### PR DESCRIPTION
**Pull request by ethervoid**
#### Summary

This PR updates the redux commit to include [this changes](https://github.com/mattermost/mattermost-redux/pull/1051)

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-22201

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

https://github.com/mattermost/mattermost-redux/pull/1051
https://github.com/mattermost/mattermost-redux/pull/1063

### CODE REVIEW

**Comment by enahum** [comment]
@ethervoid can you please resolve the conflicts?

**Comment by ethervoid** [comment]
@enahum given that this https://github.com/mattermost/mattermost-webapp/pull/4947 PR has been merged and includes a most recent commit should I point this branch to `release-5.21` and only merge it for that version?

**Comment by enahum** [comment]
@ethervoid I’ll ping you in about 1.5h and walk you through it

File: package-lock.json [hunk 1]
```diff
@@ -5657,7 +5657,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```
Comment (devinbinnie) [inline package-lock.json]: Can we remove these other changes? Just need the changes to `mattermost-redux`

**Comment by ethervoid** [comment]
@amyblais this should go in 5.21

**Pull request merged**